versioned_docs/version-v3.0/guides/assets.md

```diff
@@ -128,19 +128,19 @@ export class MyComponent {
 ```
 
 In the example above, the following allows `my-component` to display the provided asset:
-1. [`getAssetPath()`](/assets#getassetpath) is imported from `@stencil/core`
+1. [`getAssetPath()`](#getassetpath) is imported from `@stencil/core`
 2. The `my-component`'s component decorator has the `assetsDirs` property, and lists the sibling directory, `assets`. This will copy `assets` over to the distribution directory.
 3. `getAssetPath` is used to retrieve the path to the image to be used in the `<img>` tag
 
 ### Manually Moving Assets
 
-For the [dist-custom-elements](/custom-elements) output target, options like `assetsDirs` do not copy assets to the distribution directory.
+For the [dist-custom-elements](../output-targets/custom-elements.md) output target, options like `assetsDirs` do not copy assets to the distribution directory.
 
 It's recommended that a bundler (such as rollup) or a Stencil `copy` task is used to ensure the static assets are copied to the distribution directory.
 
 #### Stencil Copy Task
 
-[Stencil `copy` task](/copy-tasks)'s can be used to define files and folders to be copied over to the distribution directory.
```

Comment:
Minor, I think we dropped an apostrophe here that we didn't lose in other docs

Reply:
ah yeah, I think we want to remove that apostrophe - I'll remove it on the other files too